client/Components/Main.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Home from './Home.jsx';
+import Menu from './Hearder/Menu.jsx';
+import Footer from './Footer/Footer.jsx';
+
+import './../js/recipes.js';
+import '../Style/test.scss';
+
+/**
+ * SignIn component for user to signin
+ * @class
+ * @extends Component
+ */
+class Main extends Component {
+  constructor() {
+    super();
+  }
+  componentWillMount() {
+    const Token = localStorage.getItem('jwtToken');

```

Comment:
Don't you think `token` is better suitable as the variable name for this instead of `Token`

Reply:
I will rename it. Thanks